editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx

```diff
@@ -191,10 +194,7 @@ const InteractiveFileListItem = (props: FileListItemProps) => {
   };
 
   return (
-    <Box
-      key={props.name}
-      style={{ display: "flex", justifyContent: "space-between" }}
-    >
+    <Box style={{ display: "flex", justifyContent: "space-between" }}>
```

Comment:
I hit this issue the other day also - even if you have a unique `key` here, React still complains as it's not set at the parent level where the iteration is happening (despite it materially being the same thing).